src/main/scala/io/viash/functionality/dependencies/Dependency.scala

```diff
@@ -82,6 +82,11 @@ case class Dependency(
   @description("Location of the dependency component artifacts are written ready to be used.")
   writtenPath: Option[String] = None,
 ) {
+  if (alias.isDefined) {
+    // check functionality name
+    require(alias.get.matches("^[A-Za-z][A-Za-z0-9_]*$"), message = f"alias '${alias.get}' must begin with a letter and consist only of alphanumeric characters or underscores.")
```

Comment:
Shouldn't we also allow slashes? The dependency name for which we'll provide an alias after all is 'namespace/name'

Reply:
I would be inclined not to. The entire purpose of the alias (imo) is to assign a name which refers to the dependency inside the user-provided script.

If you include slashes or other non-alphanumerical characters, how would this be translated into code? You could replace non-alphanumerical characters with, for instance, underscores. But then why not require the user to provide an alias which does not contain non-alphanumerical characters to begin with?

Unless this alias is used somewhere else as well?

Reply:
Alright.

> how would this be translated into code?
Exactly the same way as we have to do now with the `namespace`/`name` already. Ie. replace slashes with underscores.